src/components/AddBook/AddBook.module.css

```diff
@@ -30,30 +46,36 @@
 }
 
 .form-container form input {
+  border-radius: 0.3rem;
+  border: solid 2px #e8e8e8;
```

Comment:
[Optional]
- It will be better if you create a class for most repeated rules and use that class instead of repeating same rules in different classes.